lib/mixins/acroform.js

```diff
@@ -0,0 +1,317 @@
+import { objectTypeSpreadProperty } from "@babel/types";
```

Comment:
objectTypeSpreadProperty seems not to be used

Reply:
Fixed